src/Panel/test/PanelSpec.js

```diff
@@ -49,18 +47,14 @@ describe('Panel', () => {
     assert.equal(instance.querySelector('.rs-panel-body').getAttribute('role'), 'button');
   });
 
-  it('Should call onSelect callback', done => {
-    const doneOp = eventKey => {
-      if (eventKey === 12) {
-        done();
-      }
-    };
-
+  it('Should call onSelect callback', () => {
+    const onSelectSpy = sinon.spy();
     const instance = getDOMNode(
-      <Panel collapsible onSelect={doneOp} eventKey={12} header={'abc'} />
+      <Panel collapsible onSelect={onSelectSpy} eventKey={12} header={'abc'} />
     );
 
     ReactTestUtils.Simulate.click(instance.querySelector('.rs-panel-header'));
+    assert.equal(onSelectSpy.firstCall.firstArg, 12);
```

Comment:
断言函数调用传参可以使用 `.calledWith` 方法

```suggestion
    assert(onSelectSpy.calledWith(12));
```

特别是当断言多个参数时，表义更加清晰。

Reply:
我反而习惯 equal 这种判断表达式的断言，让人更清晰一些。

Reply:
那样的话最好先断言一次 `.called`，不然 `firstCall` 有可能是 `null`。

Reply:
嗯嗯

Reply:
或者也许你会喜欢 BDD 风格的断言 :)

```suggestion
    expect(onSelectSpy).to.have.been.calledWith(12);
```